waspc/src/Lib.hs

```diff
@@ -24,8 +24,14 @@ compile waspFile outDir options = do
 
     case Parser.parseWasp waspStr of
         Left err -> return $ Left (show err)
-        Right wasp -> do
-            externalCodeFiles <- ExternalCode.readFiles (CompileOptions.externalCodeDirPath options)
-            generateCode $ wasp `setExternalCodeFiles` externalCodeFiles
+        Right wasp -> expandWaspASTWithCompileOptions wasp options >>= generateCode
   where
     generateCode wasp = Generator.writeWebAppCode wasp outDir options >> return (Right ())
+    
+expandWaspASTWithCompileOptions :: Wasp -> CompileOptions -> IO Wasp
```

Comment:
Ok this part is slightly weird.
You say you expand it with CompileOptions, but you also add external files -> so the name is not completely correct.
Also, I find it unusual that you are replicating stuff from compile options in Wasp -> why do we need to have isBuild in both places? Why not just add CompileOptions as one of the properties of Wasp, for now?

Reply:
resolved, changed the name to better reflect that not only copying is happening.